app/viewmodels/IndexPageHelper.scala

```diff
@@ -65,51 +66,60 @@ class IndexPageHelper @Inject()(p: views.html.components.p,
           )),
           p(content = stringAsHtml(
             messages("lsp.pointSummary.penaltyPoints.overview.whatHappensWhenThresholdExceeded", threshold)
-          ))
+          )),
+          getGuidanceLink
         )
-        if(currentPoints == threshold - 1) {
+        if (currentPoints == threshold - 1) {
           html(base.+:(warningText(stringAsHtml(messages("lsp.pointSummary.penaltyPoints.overview.warningText")))): _*)
         } else {
           html(base: _*)
         }
       }
 
-      case (currentPoints, threshold, _) if currentPoints < threshold-1 => {
-          html(
-            renderPointsTotal(currentPoints),
-            p(content = getPluralOrSingularContentForOverview(currentPoints, etmpData.lateSubmissions)),
-            p(content = stringAsHtml(
-              messages("lsp.pointSummary.penaltyPoints.overview.anotherPoint")
-            )),
-            p(content = stringAsHtml(
-              messages("lsp.pointSummary.penaltyPoints.overview.whatHappensWhenThresholdExceeded", threshold)
-            ))
-          )
+      case (currentPoints, threshold, _) if currentPoints < threshold - 1 => {
+        html(
+          renderPointsTotal(currentPoints),
+          p(content = getPluralOrSingularContentForOverview(currentPoints, etmpData.lateSubmissions)),
+          p(content = stringAsHtml(
+            messages("lsp.pointSummary.penaltyPoints.overview.anotherPoint")
+          )),
+          p(content = stringAsHtml(
+            messages("lsp.pointSummary.penaltyPoints.overview.whatHappensWhenThresholdExceeded", threshold)
+          )),
+          getGuidanceLink
+        )
       }
-      case (currentPoints, threshold, _) if currentPoints == threshold-1 => {
+      case (currentPoints, threshold, _) if currentPoints == threshold - 1 => {
         html(
           renderPointsTotal(currentPoints),
           warningText(stringAsHtml(messages("lsp.pointSummary.penaltyPoints.overview.warningText"))),
-          p(getPluralOrSingularContentForOverview(currentPoints, etmpData.lateSubmissions))
+          p(getPluralOrSingularContentForOverview(currentPoints, etmpData.lateSubmissions)),
+          getGuidanceLink
         )
       }
       //TODO: replace this with max scenarios - added this so we don't get match errors.
-      case _ => {
-        p(content = html(stringAsHtml("")))
+      case (currentPoints, threshold, _) if currentPoints >= threshold => {
```

Comment:
Checked the precedence - I think this case statement should be put below the first case statement.